content/02.introduction.md

```diff
@@ -3,4 +3,70 @@ Set introduction for paper here
 Talk about problem, goal, and significance of paper
 
 ## Recent Work
-Talk about what has been done in the field in regards to text mining and knowledge base integration
+Relationship extraction is the process of detecting and classifying semantic relationships from a set of text.
```

Comment:
"set of text" sounds a bit odd, maybe "corpus of texts" or "collection of text" or just "text".